infrastructure/scripts/provision/core.sh

```diff
@@ -12,20 +11,19 @@ source ${INFRASTRUCTURE_PATH}/scripts/lib/k8s.sh
 cd ${INFRASTRUCTURE_PATH}/scripts/
 wait-for-service-account
 
-
 echo "Deploying the Airy Core Platform with the ${AIRY_VERSION} image tag"
 
 if [[ -f ${INFRASTRUCTURE_PATH}/airy.yaml ]]; then
     yq eval '.global.appImageTag="'${AIRY_VERSION}'"' -i ${INFRASTRUCTURE_PATH}/airy.yaml
-    helm install core ${INFRASTRUCTURE_PATH}/helm-chart/ --values ${INFRASTRUCTURE_PATH}/airy.yaml --timeout 1000s > /dev/null 2>&1
-    wget -qnv https://airy-core-binaries.s3.amazonaws.com/alpine/airy.gz
+    helm install core ${INFRASTRUCTURE_PATH}/helm-chart/ --values ${INFRASTRUCTURE_PATH}/airy.yaml --set global.ngrokEnabled=${NGROK_ENABLED} --timeout 1000s > /dev/null 2>&1
+    wget -qnv https://airy-core-binaries.s3.amazonaws.com/${AIRY_VERSION}/alpine/amd64/airy.gz
```

Comment:
@pascal-airy do you think this will solve https://github.com/airyhq/airy/issues/739 ? 
I created a `develop` directory, where we would need to put the new binary when we merge to develop (if the cli has changed)

Reply:
yeah but we should find out how we can build the alpine version with bazel and then the  develop will automatically go into the "*-alpha" folder

Reply:
Yes, that we must solve anyway, agreed :+1: But I think we must have the folder `develop`, otherwise we should push the binaries with the suffix, to a future-release-folder? Or is there a `future-release-alpha` folder created by default?